src/Hedgehog/Property.fs

```diff
@@ -56,15 +56,18 @@ module Property =
             failure
 
     let counterexample (msg : unit -> string) : Property<unit> =
-        Gen.constant (Journal.singleton msg, Success ()) |> ofGen
+        (Journal.singleton msg, Success ()) |> Gen.constant |> ofGen
 
     let private mapGen
             (f : Gen<Journal * Outcome<'a>> -> Gen<Journal * Outcome<'b>>)
-            (x : Property<'a>) : Property<'b> =
-        toGen x |> f |> ofGen
+            (p : Property<'a>) : Property<'b> =
+        p |> toGen |> f |> ofGen
 
     let map (f : 'a -> 'b) (x : Property<'a>) : Property<'b> =
-        (mapGen << GenTuple.mapSnd << Outcome.map) f x
+        f |> Outcome.map |> GenTuple.mapSnd |> mapGen <| x
```

Comment:
The `<|` here should be removed, since we went through the trouble of removing the others a while back 😄

Reply:
Done